content/cpp/concepts/math-functions/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'sqrt()'
+Description: 'Returns square root of the argument'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Returns square root of the argument
+
+## Syntax
+
+```cpp
+ sqrt(x)
+```
+
+## Example
+
+use `sqrt()` to find the square root of `4`:
+
+```cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+  double x = 4;
+  double result;
+
+  result = std::sqrt(x);
+
+  std::cout << "The square root of " << x << " is " << result << "\n";
+  // Output: The square root of 4 is 2
+```
+
+## Codebyte Example
+
+```codebyte/cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+  double x = 9;
+  double result;
+
+  result = std::sqrt(x);
+
+  std::cout << "The square root of " << x << " is " << result << "\n";
+  // Output: The square root of 9 is 3
```

Comment:
```suggestion
  std::cout << "The square root of " << x << " is " << result << "\n";
```

Since this is part of a runnable Codebyte example, the output will be rendered so there is no need for the comment.